app/src/main/kotlin/com/wire/android/ui/debugscreen/DebugScreen.kt

```diff
@@ -168,18 +174,29 @@ fun LoggingSection(
     TextRowItem(
         "Share Logs",
         trailingIcon = android.R.drawable.ic_menu_share
-    ) { context.startMultipleFileSharingIntent(logFilePath()) }
+    ) {
+        val dir = File(logFilePath()).parentFile
+        val fileUris = context.getUrisOfFilesInDirectory(dir)
+        val intent = context.multipleFileSharingIntent(fileUris)
+        // The first log file is simply text, not compressed. Get its mime type separately
+        // and set it as the mime type for the intent.
```

Comment:
sorry here I didn't understand what we are trying to achieve, is it to make the shared files get the correct mimetype ?
if that's the case the original startMultipleFileSharingIntent function was doing that already. but in this case so we can just update that function to make it return intent so we can use it in the case of general share or the email share only below.

Reply:
Replied privately